core/ledger/kvledger/txmgmt/privacyenabledstate/test_exports.go

```diff
@@ -153,10 +156,15 @@ func (env *CouchDBCommonStorageTestEnv) Init(t testing.TB) {
 		stateDBConfig,
 		[]string{"lscc", "_lifecycle"},
 	)
+	logger.Infof("processing")
```

Comment:
do we need this?

also, as a general Go practice, err check should come immediately after where err might be produced. let's not insert code in between.